src/scaffoldmaker/utils/annulusmesh.py

```diff
@@ -6,8 +6,9 @@
 import copy
 import math
 from opencmiss.utils.zinc.field import findOrCreateFieldCoordinates
+from opencmiss.utils.zinc.general import ChangeManager
 from opencmiss.zinc.element import Element
-from opencmiss.zinc.field import Field
+from opencmiss.zinc.field import Field, FieldGroup
 from opencmiss.zinc.node import Node
```

Comment:
Remove the remnants to your changes to annulus mesh since you've moved element/node deletion code elsewhere.